git-find-lfs-extensions.py

```diff
@@ -111,3 +123,10 @@ def print_line(ext, share_large, count_large, count_all, size_all, min, max):
             int(result[ext]['min']),
             int(result[ext]['max'])
         )
+
+print("\nAdd to .gitattributes:\n")
+for ext in sorted(result, key=lambda x: (result[x]['type'], x)):
+    if len(ext) > 0 and result[ext]['type'] == "binary" and result[ext]['count_large'] > 0:
+        print('*.{} filter=lfs diff=lfs merge=lfs -text'.format(
+            "".join(c if ('0' <= c <= '9') else "[" + c.upper() + c.lower() + "]" for c in ext)
+        ))
```

Comment:
Very nice! All of it. 👍 

A few minor nits:
* the git attributes are created case insensitive but the file extension in the table are not. Maybe we should make all filenames lower case in table?
* you exclude numbers in your wild card builder. shouldn't we exclude special characters (e.g. `[`) as well? maybe we should only include letters? Some special characters need to be escaped to (e.g. `[` must be `\[`)